test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -13,6 +13,160 @@ namespace NuGet.CommandLine.Test
     public class NuGetPackCommandTest
     {
         [Fact]
+        public void PackCommand_IncludeExcludePackageFromNuspec()
+        {
+            var nugetexe = Util.GetNuGetExePath();
+            var workingDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
```

Comment:
C:\nuget\NuGet.Client\test\NuGet.Core.Tests\NuGet.Configuration.Test\TestUtils.cs for example